.github/workflows/git-artifacts.yml

```diff
@@ -387,13 +387,6 @@ jobs:
         with:
           name: arm64-artifacts
           path: ${{github.workspace}}/arm64
-      # Workaround for Git Credential Manager Core on ARM64: https://github.com/git-for-windows/git/issues/3015
-      - name: Create git-credential-manager-core wrapper for ARM64
-        if: matrix.arch.arm64 == true
-        shell: bash
-        run: |
-          printf '%s\n' '#!/bin/sh' 'exec /mingw32/libexec/git-core/git-credential-manager-core.exe "$@"' > arm64/bin/git-credential-manager-core
```

Comment:
I bet it works if it's put into `arm64/libexec/git-core/` instead of `arm64/bin/`.

And honestly, I'd prefer that over the `build-extra` PR, to minimize the impact outside `git-artifacts`...

Reply:
@dscho that works, thanks! Just updated the commit accordingly.

I was curious as to what `libexec` does exactly, and [found this definition](https://sny.no/2017/05/libexec):

> libexec is meant for system daemons and system utilities executed by other programs. That is, the binaries put in this namespaced directory are meant for the consumption of other programs, and are not intended to be executed directly by users.

Learning every day 🚀